recipes/apriltag/all/patches/fix-cmake-3.1.1.patch

```diff
@@ -10,7 +10,8 @@
  aux_source_directory(common COMMON_SRC)
  aux_source_directory(. APRILTAG_SRCS)
 @@ -10,7 +8,8 @@
- set(CMAKE_BUILD_TYPE Release)
+-set(CMAKE_BUILD_TYPE Release)
++set(CMAKE_BUILD_TYPE Release)
```

Comment:
Looks out of place, it doesn't change anything

Reply:
Assuming this should patch the other patches

```suggestion
+#set(CMAKE_BUILD_TYPE Release)
```

Reply:
You're right. Thanks for fixing the typo.